Changelog.md

```diff
@@ -5,6 +5,9 @@ Changelog
 
 * Use ceil in `ResettingController` for a better token lifetime approximation
 * Removed unused translation keys
+* Removed form deprecations
+* Added `symfony/templating` as a dependency
```

Comment:
it was already required previously, due to the FrameworkBundle requirement before 3.1. So nothing has changed.
and the change to the native Twig notation actually means we could probably drop the requirement